lib/web_ui/lib/src/engine/surface/recording_canvas.dart

```diff
@@ -440,19 +440,22 @@ class RecordingCanvas {
         return;
       }
     }
-    _hasArbitraryPaint = true;
-    _didDraw = true;
-    ui.Rect pathBounds = path.getBounds();
-    final double paintSpread = _getPaintSpread(paint);
-    if (paintSpread != 0.0) {
-      pathBounds = pathBounds.inflate(paintSpread);
+    SurfacePath sPath = path;
+    if (sPath.subpaths.isNotEmpty) {
```

Comment:
Nice! I think I saw this in the cupertino slider as well. There were some commands thrown into the canvas without anything actually painted.